squirrels/_api_routes/auth.py

```
@@ -183,59 +247,60 @@ async def logout(request: Request):
-        # API Key endpoints
-        api_key_path = '/api-keys'
-
-        class ApiKeyRequestBody(BaseModel):
-            title: str = Field(description=f"The title of the API key")
-            expiry_minutes: int | None = Field(
-                default=None, 
-                description=f"The number of minutes the API key is valid for (or valid indefinitely if not provided)."
-            )
+        if not is_external:
+            # Change password endpoint
+            change_password_path = '/password'
```

Ruff (0.14.13)

Possible hardcoded password assigned to: "change_password_path"

(S105)